futurenhs.app/components/_contentBlockComponents/TextContentBlock/index.tsx

```diff
@@ -84,17 +85,7 @@ export const TextContentBlock: (props: Props) => JSX.Element = ({
 
     return (
         <div id={blockId} className={generatedClasses.wrapper}>
-            {title &&
-                <Heading
-                    headingLevel={headingLevel}
-                    className={generatedClasses.heading}
-                >
-                    {title}
-                </Heading>
-            }
-            {mainText && 
-                <RichText bodyHtml={mainText} wrapperElementType="div" />
-            }
+            <Image src="/images/example-1.jpg" layout="responsive" width={1920} height={1200} />
```

Comment:
Perhaps I'm missing something but is this intentional? Looks like it replaces the text block with an image?

Reply:
Ooops! Good spot! I was tinkering with something to demo some screenshots for an upcoming story. Shouldn't have committed that!